guide/contributing/codebase-overview.md

```diff
@@ -29,22 +29,29 @@ We utilize the following tools for the development, release, and distribution pr
   manager for its speed and deep dependency version locking
 * [Openshift](https://www.openshift.com/) and [Docker](https://www.docker.com/): the CI pipeline is largely
   based on the isomorphic stater kit pipeline, using Docker as the build artifact
+* [Lerna](https://lernajs.io/): A tool for managing the repository that consists of multiple packages.
```

Comment:
We could also add Yarn Workspaces, though it's not a priority.

Reply:
I'd probably just copy the description of lerna straight from their website: "A tool for managing JavaScript projects with multiple packages."